### PR TITLE
[IR-3427] Unit Test - Spatial XR coverage: Functions & Static Data

### PR DESCRIPTION
**Pull request by heysokam**
## References
https://tsu.atlassian.net/browse/IR-3427
Split from: https://tsu.atlassian.net/browse/IR-5704

### CODE REVIEW

File: packages/spatial/src/xr/XRSessionFunctions.test.ts [hunk 1]
```diff
@@ -325,16 +333,155 @@ describe('onSessionEnd', () => {
...
+  })
+}) //:: getReferenceSpaces
+
+describe.only('setupXRSession', () => {
+  beforeEach(async () => {
+    createEngine()
+    mockSpatialEngine()
+    WebXREventDispatcher.instance.dispatchEvent({
+      type: POLYFILL_ACTIONS.DEVICE_INIT,
+      detail: { stereoEffect: false, device: DeviceDefinitions.Default }
```
Comment (HexaField) [inline packages/spatial/src/xr/XRSessionFunctions.test.ts]: `device` should be `deviceDefinition`

File: packages/spatial/tests/webxr/emulator.ts [hunk 2]
```diff
@@ -57,14 +58,30 @@ export function getLastXRSessionData(session: XRSession | null) {
...
+
 /**
  * @description Requests an emulated XRSession.
  * @why Shorthand for initializing an emulated XRSession from unit tests
  * */
 export async function requestEmulatedXRSession(deviceDefinition = DeviceDefinitions.Default) {
   WebXREventDispatcher.instance.dispatchEvent({
     type: POLYFILL_ACTIONS.DEVICE_INIT,
-    detail: { stereoEffect: false, deviceDefinition }
+    detail: { stereoEffect: false, device: deviceDefinition }
```
Comment (HexaField) [inline packages/spatial/tests/webxr/emulator.ts]: `device` should be `deviceDefinition`

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/spatial/src/xr/XRSessionFunctions.test.ts (hunk 1 above)
Comment (heysokam) [reply packages/spatial/src/xr/XRSessionFunctions.test.ts]: Good catch, thanks!
Fixed with 81bce7e0aa87a77edee4ea2238475a826b52661e

File: packages/spatial/tests/webxr/emulator.ts (hunk 2 above)
Comment (heysokam) [reply packages/spatial/tests/webxr/emulator.ts]: Fixed with 81bce7e0aa87a77edee4ea2238475a826b52661e

**Review by HexaField — APPROVED** [review]

**Pull request merged**